lib/addons/src/index.js

```diff
@@ -2,7 +2,7 @@ export class AddonStore {
   constructor() {
     this.loaders = {};
     this.panels = {};
-    this.channel = null;
+    this.channel = { on() {}, emit() {} };
```

Comment:
Hmm, I think this change is going to just make it harder to debug problems with multiple loading. Why did you add it @ndelangen ?